test_pull_requests

```diff
@@ -1084,8 +1206,7 @@ popd
           #    - NOTFOUND
 
           # Capture the build_url from the regex match
-          build_url = $1
-          extended_tests = $2
+          _, build_url, base_commit, extended_tests = extract_bot_comment_metadata(comment, settings)
```

Comment:
It's weird to be in a case statement for a regex and reparse one of the values the outer regex is already capturing.